src/components/Dialog/Dialog.scss

```diff
@@ -1,4 +1,8 @@
 .wkp-dialog {
+  .tm-rnd {
+    display: flex;
```

Comment:
useless becaus it's overrided by the style attribute

Reply:
Lint won't accept that, for descending specificity